strr-web/pages/registration-details/[id]/index.vue

```diff
@@ -85,21 +93,40 @@
                 {{ application?.unitAddress.postalCode || '-' }}
               </p>
               <p>
-                {{ application?.unitAddress.country ? regionNamesInEnglish.of(application?.unitAddress.country) : '-' }}
+                {{ application?.unitAddress.country
+                  ? regionNamesInEnglish.of(application?.unitAddress.country)
+                  : '-' }}
               </p>
             </BcrosFormSectionReviewItem>
+            <BcrosFormSectionReviewItem
+              :title="tApplicationDetails('parcelIdentifier')"
+              :content="application?.unitDetails.parcelIdentifier || '-'"
+              data-test-id="parcel-identifier"
+              class="break-all"
+            />
             <BcrosFormSectionReviewItem
               :title="tApplicationDetails('businessLicense')"
               :content="application?.unitDetails.businessLicense || '-'"
+              data-test-id="business-license"
               class="break-all"
             />
+            <BcrosFormSectionReviewItem :title="tApplicationDetails('listingLink')">
+              <a
+                :href="application?.listingDetails[0].url"
```

Comment:
I had issues with this, when there are no listing links on Application Details page, the page will not load. Need to check if `listingDetails` is not empty.

Reply:
do you want me to do that in this PR or just a point to note here?

Reply:
Yea, please fix it here.

Reply:
From the discussion earlier, you said you a busy today, so I can fix it.